jcl/jpp_configuration.xml

```diff
@@ -83,7 +83,30 @@
 		<parameter name="msg:outputdir" value="com/ibm/oti/util"/>
 		<parameter name="jxerules:outputdir" value="java/lang"/>
 	</configuration>
-
+	
+	<configuration
+		  label="SIDECAR18-SE-OPENJ9"
+		  outputpath="SIDECAR18-SE-OPENJ9/src"
+		  flags="Sidecar18-SE-OpenJ9,CUDA4J"
+		  dependencies="SIDECAR18-SE"
+		  jdkcompliance="1.8">
+		<classpathentry kind="lib" path="/binaries/vm/third/rt-compressed.sun180.jar"/>
```

Comment:
This relies on a binary file - a stripped down version of rt.jar - which isn't available here.  Can it be based on a unmodified rt.jar?  Or can we produce a script that knows how to strip rt.jar?

Reply:
It can (now) be based on an unmodified rt.jar from Java 8, plus the tools.jar. There is no need to use a stripped down rt.jar. The modified version was created only to save repo & disk space, and build time. As this part of the preprocessor configuration is only used by the Eclipse Java Preprocessor plugin, which at this time isn't contributed to open source. I don't see any reason to change from the existing classpathentry, or what I would change it to.